compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -170,16 +155,15 @@ impl Visibility {
 
 /// LLVMUnnamedAddr
 #[repr(C)]
-pub enum UnnamedAddr {
-    No,
-    Local,
-    Global,
```

Comment:
For enums that are passed across FFI to C/C++ code, we shouldn't delete Rust-side variants that are unused.

Doing so just makes it harder to verify that the Rust-side enum and the C-side enum have the same layout, which is essential for soundness.